external-api/src/constants/eStore.constant.ts

```diff
@@ -9,3 +9,12 @@ export const ESTORE_CRON_STATUS = {
   COMPLETED: 'Completed',
   FAILED: 'Failed',
 };
+
+export const ENDPOINT = {
```

Comment:
Just suggestion. Looks like these are strings used in URL to construct `Endpoint`. Not sure if this change makes endpoints easier to read.

Maybe constant can be renamed or values made longer, like BUYER value could be `sites/:siteId/buyer`